README.md

```diff
@@ -32,6 +32,30 @@ You can install peepDB directly from PyPI:
 pip install peepdb
 ```
 
+### Docker
+
+peepDB is also available as a `docker` image
+
+```
+docker pull [docker repo]/peepdb:0.1.4
```

Comment:
@Aherontas  I'm not sure what docker repo you plan to use, can update this accordingly if you let me know what it should be